deploy.sh

```diff
@@ -3,11 +3,5 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-MONORAIL_KEY="monorail-key.json"
-if [[ ! -f "$MONORAIL_KEY" ]]; then
-    echo "$MONORAIL_KEY needed to deploy"
-    exit 1
-fi
-
 VERSION="$(git describe --always --dirty --long)"
-gcloud app deploy --project=ecosystem-infra-rotation --version="$VERSION"
+gcloud app deploy --no-promote --project=ecosystem-infra-rotation --version="$VERSION"
```

Comment:
Given `--no-promote` here, can you update the README to also say how to deploy to https://ecosystem-infra-rotation.appspot.com/?